inspire/base/templates/format/record/Experiment_HTML_detailed.tpl

```diff
@@ -0,0 +1,129 @@
+{#
+## This file is part of INSPIRE.
+## Copyright (C) 2015 CERN.
+##
+## INSPIRE is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## INSPIRE is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{% extends "format/record/Default_HTML_detailed.tpl" %}
+
+{% block header %}
+  <div class="row">
+    <div class="col-md-12">
+      <h3>
+      {% for collection in record['collections'] %}
+        {% if 'primary' in collection %}
+          <span class="label label-default pull-left {% if not loop.first %} collection-primary {% endif %}">
+          {{ collection['primary'] }}</span>
+        {% endif %}
+      {% endfor %}
+      </h3>
+    </div>
+  </div>
+  <div class="row"><div class="col-md-12"><p></p></div></div>
+  <div class="row"><div class="col-md-12"><p></p></div></div>
+  <div class="row">
+    <div class="col-md-12">
+     <div class="record-title pull-left">
+        {{ record['experiment_name'][0] }}
+        {% if record['affiliation'] %}
+          ({{ record['affiliation'][0] }})
+        {% endif %}
+      </div>
+    </div>
+  </div>
+  {% if record['breadcrum_title'] %}
+    <div class="row">
+      <div class="col-md-12">
+        <div class="pull-left">
+          {{ record['breadcrum_title'] }}
+        </div>
+      </div>
+    </div>
+  {% endif %}
+  <div class="row"><div class="col-md-12"><p></p></div></div>
+  <div class="row"><div class="col-md-12"><p></p></div></div>
+  <div class="row">
+    <div class="col-md-12">
+      <div class="pull-left">
+        (Started: 2004, Still Running)
```

Comment:
Currently yes, because it is missing from the dojson. I can add it but it is also missing from the experiments.json. With what name should I add it?